### PR TITLE
Update NuGet.Server docs to talk about the new push URL

### PR DESCRIPTION
**Pull request by joelverhagen**
Address https://github.com/NuGet/docs.microsoft.com-nuget/issues/1038.

### CODE REVIEW

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[cbf1227](https://github.com/NuGet/docs.microsoft.com-nuget/commits/cbf1227d91096ccd3de31aa4ea79bcb6b58c5e19)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/hosting-packages/NuGet-Server.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/NuGet-Server.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_00-NuGet.Server-ProjectType.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_00-NuGet.Server-ProjectType.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_01-NuGet.Server-Set4.6.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_01-NuGet.Server-Set4.6.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_02-NuGet.Server-Empty.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_02-NuGet.Server-Empty.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_02-NuGet.Server-Package.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_02-NuGet.Server-Package.png) | :white_check_mark:Succeeded | |
[docs/hosting-packages/media/Hosting_03-NuGet.Server-Package-Folder.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_03-NuGet.Server-Package-Folder.png) | :white_check_mark:Succeeded | |
[docs/hosting-packages/media/Hosting_03-NuGet.Server-Package.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_03-NuGet.Server-Package.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_04-NuGet.Server-FeedHomePage.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_04-NuGet.Server-FeedHomePage.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_05-NuGet.Server-Package-Folder.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_05-NuGet.Server-Package-Folder.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/media/hosting_05-nuget.server-package-folder.png?branch=pr-en-us-1935) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C3%5C3%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202003030001316420-1935%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=UMzxKPZMZjmv5%2BktD3l%2FBpPsc5PQpdQmSV1XztAp7sI%3D&st=2020-03-02T23%3A57%3A17Z&se=2020-04-03T00%3A02%3A17Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[93cd48a](https://github.com/NuGet/docs.microsoft.com-nuget/commits/93cd48a3edd00035e88d2d824466556a2bb742c2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/hosting-packages/NuGet-Server.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/NuGet-Server.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_00-NuGet.Server-ProjectType.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_00-NuGet.Server-ProjectType.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_01-NuGet.Server-Set4.6.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_01-NuGet.Server-Set4.6.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_02-NuGet.Server-Empty.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_02-NuGet.Server-Empty.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_02-NuGet.Server-Package.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_02-NuGet.Server-Package.png) | :white_check_mark:Succeeded | |
[docs/hosting-packages/media/Hosting_03-NuGet.Server-Package-Folder.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_03-NuGet.Server-Package-Folder.png) | :white_check_mark:Succeeded | |
[docs/hosting-packages/media/Hosting_03-NuGet.Server-Package.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_03-NuGet.Server-Package.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_04-NuGet.Server-FeedHomePage.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_04-NuGet.Server-FeedHomePage.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_05-NuGet.Server-Package-Folder.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_05-NuGet.Server-Package-Folder.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/media/hosting_05-nuget.server-package-folder.png?branch=pr-en-us-1935) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C3%5C3%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202003030006227756-1935%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=S3fsAVoMmZSDF6fIQZlOXDhX%2BtCo6%2BKzUzmyCfVsOVc%3D&st=2020-03-03T00%3A02%3A09Z&se=2020-04-03T00%3A07%3A09Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/hosting-packages/NuGet-Server.md [hunk 1]
```diff
@@ -97,12 +82,45 @@ To enable this capability, set the `apiKey` to a value (ideally a strong passwor
...
+
+`packagesPath` can be an absolute or virtual path.
+
+When `packagesPath` is omitted or left blank, the packages folder is the default `~/Packages`.
+
+## Making packages available when you publish the web app
+
+To make packages available in the feed when you publish the application to a server, add each `.nupkg` files to the `Packages` folder in Visual Studio, then set each one's **Build Action** to **Content** and **Copy to Output Directory** to **Copy always**:
+
+    ![Copying packages to the Packages folder in the project](media/Hosting_05-NuGet.Server-Package-Folder.png)
```
Comment (loic-sharma) [inline docs/hosting-packages/NuGet-Server.md]: This seems to be broken, see: https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935#making-packages-available-when-you-publish-the-web-app

File: docs/hosting-packages/NuGet-Server.md [hunk 2]
```diff
@@ -97,12 +82,45 @@ To enable this capability, set the `apiKey` to a value (ideally a strong passwor
...
+
 ## Removing packages from the feed
 
 With NuGet.Server, the [nuget delete](../reference/cli-reference/cli-ref-delete.md) command removes a package from the repository provided that you include the API key with the comment.
 
 If you want to change the behavior to delist the package instead (leaving it available for package restore), change the `enableDelisting` key in `web.config` to true.
 
+## Configuring the Packages folder
+
+With `NuGet.Server` 1.5 and later, you can more specifically configure the package folder using the `appSetting/packagesPath` value in `web.config`:
```
Comment (loic-sharma) [inline docs/hosting-packages/NuGet-Server.md]: What do you think of this?

```suggestion
With `NuGet.Server` 1.5 and later, you can customize the package folder using the `appSetting/packagesPath` value in `web.config`:
```

Comment (joelverhagen) [reply docs/hosting-packages/NuGet-Server.md]: Sounds good to me

File: docs/hosting-packages/NuGet-Server.md (hunk 1 above)
Comment (joelverhagen) [reply docs/hosting-packages/NuGet-Server.md]: Oh nice catch, I'll fix that

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[b48e54a](https://github.com/NuGet/docs.microsoft.com-nuget/commits/b48e54a2ca64c67be6d3f8ba0882106bda0890d1)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/hosting-packages/NuGet-Server.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/NuGet-Server.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_00-NuGet.Server-ProjectType.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_00-NuGet.Server-ProjectType.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_01-NuGet.Server-Set4.6.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_01-NuGet.Server-Set4.6.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_02-NuGet.Server-Empty.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_02-NuGet.Server-Empty.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_02-NuGet.Server-Package.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_02-NuGet.Server-Package.png) | :white_check_mark:Succeeded | |
[docs/hosting-packages/media/Hosting_03-NuGet.Server-Package-Folder.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_03-NuGet.Server-Package-Folder.png) | :white_check_mark:Succeeded | |
[docs/hosting-packages/media/Hosting_03-NuGet.Server-Package.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_03-NuGet.Server-Package.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_04-NuGet.Server-FeedHomePage.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_04-NuGet.Server-FeedHomePage.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_05-NuGet.Server-Package-Folder.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_05-NuGet.Server-Package-Folder.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/media/hosting_05-nuget.server-package-folder.png?branch=pr-en-us-1935) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C3%5C3%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202003031847043163-1935%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=cGvhy%2BFiAGrKJQByiG4uMh0IAdybsigoC6ioDg6M2yo%3D&st=2020-03-03T18%3A42%3A48Z&se=2020-04-03T18%3A47%3A48Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[c0c7e2e](https://github.com/NuGet/docs.microsoft.com-nuget/commits/c0c7e2e0f8d9c53198711c69196dbde49ca2b859)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/hosting-packages/NuGet-Server.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/NuGet-Server.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_00-NuGet.Server-ProjectType.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_00-NuGet.Server-ProjectType.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_01-NuGet.Server-Set4.6.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_01-NuGet.Server-Set4.6.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_02-NuGet.Server-Empty.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_02-NuGet.Server-Empty.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_02-NuGet.Server-Package.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_02-NuGet.Server-Package.png) | :white_check_mark:Succeeded | |
[docs/hosting-packages/media/Hosting_03-NuGet.Server-Package-Folder.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_03-NuGet.Server-Package-Folder.png) | :white_check_mark:Succeeded | |
[docs/hosting-packages/media/Hosting_03-NuGet.Server-Package.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_03-NuGet.Server-Package.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_04-NuGet.Server-FeedHomePage.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_04-NuGet.Server-FeedHomePage.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_05-NuGet.Server-Package-Folder.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_05-NuGet.Server-Package-Folder.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C3%5C3%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202003031850341498-1935%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=JtLCSDuQf%2FF80lo9cWL7%2BT01dhD39Ui7WfXF87WCke0%3D&st=2020-03-03T18%3A46%3A13Z&se=2020-04-03T18%3A51%3A13Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by karann-msft** [comment]
Docs Build status updates of commit _[0869bb4](https://github.com/NuGet/docs.microsoft.com-nuget/commits/0869bb41c9563526bc494962dc009614b600f260)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/hosting-packages/NuGet-Server.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/NuGet-Server.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_00-NuGet.Server-ProjectType.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_00-NuGet.Server-ProjectType.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_01-NuGet.Server-Set4.6.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_01-NuGet.Server-Set4.6.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_02-NuGet.Server-Empty.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_02-NuGet.Server-Empty.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_02-NuGet.Server-Package.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_02-NuGet.Server-Package.png) | :white_check_mark:Succeeded | |
[docs/hosting-packages/media/Hosting_03-NuGet.Server-Package-Folder.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_03-NuGet.Server-Package-Folder.png) | :white_check_mark:Succeeded | |
[docs/hosting-packages/media/Hosting_03-NuGet.Server-Package.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_03-NuGet.Server-Package.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_04-NuGet.Server-FeedHomePage.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_04-NuGet.Server-FeedHomePage.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |
[docs/hosting-packages/media/Hosting_05-NuGet.Server-Package-Folder.png](https://github.com/NuGet/docs.microsoft.com-nuget/blob/jver-1038/docs/hosting-packages/media/Hosting_05-NuGet.Server-Package-Folder.png) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/nuget/hosting-packages/nuget-server?branch=pr-en-us-1935) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2020%5C3%5C3%5C19a7edc0-f91d-0fd5-6e5f-4d131db05c11%5CPullRequest%5C202003031857075989-1935%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=ciWiRifGnYsmUJ%2FAuh8qZfeJ0v%2FTG6CrYqJCcCjxBIc%3D&st=2020-03-03T18%3A52%3A53Z&se=2020-04-03T18%3A57%3A53Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**